setup.cfg

```diff
@@ -34,7 +34,7 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .tox, .git, __pycache__, .ipynb_checkpoints
-ignore = W503
+extend-ignore = W503, SFS3 # String literal formatting using f-string.
```

Comment:
So just to clarify, by ignoring `SFS3`, which catches f-string formatting, we are only accepting f-string formatting?

Reply:
Yep, that's correct.